tests/lib.rs

```diff
@@ -193,23 +193,25 @@ fn cleanup_file(f: &str) {
     }
 }
 
-fn control_fdr(test: &str, event_str: &str, alpha: f64, local: bool) {
+fn control_fdr(test: &str, events: Vec<&str>, alpha: f64, local: bool) {
     let basedir = basedir(test);
     let output = format!("{}/calls.filtered.bcf", basedir);
     cleanup_file(&output);
-    varlociraptor::filtration::fdr::control_fdr(
-        &format!("{}/calls.matched.bcf", basedir),
-        Some(&output),
-        &[varlociraptor::SimpleEvent {
-            name: event_str.to_owned(),
-        }],
-        Some(&varlociraptor::variants::model::VariantType::Deletion(
-            Some(1..30),
-        )),
-        LogProb::from(Prob(alpha)),
-        local,
-    )
-    .unwrap();
+    for event_str in events {
+        varlociraptor::filtration::fdr::control_fdr(
+            &format!("{}/calls.matched.bcf", basedir),
+            Some(&output),
+            &[varlociraptor::SimpleEvent {
```

Comment:
The events have to be collected into this argument here instead of calling control_fdr multiple times.